sortinghat/core/db.py

```diff
@@ -1125,3 +1125,86 @@ def _handle_integrity_error(model, exc):
         eid = m.group('value').split('-')[0]
 
     raise AlreadyExistsError(entity=entity, eid=eid)
+
+
+def move_domain(trxl, domain, organization):
+    """Move a domain to an organization
+
+    Shifts the given `domain` object to the given `organization`.
+
+    As a result, it returns the `Domain` object with the updated data.
+
+    :param trxl: TransactionsLog object from the method calling this one
+    :param domain: domain to be moved
+    :param organization: organization to move the domain to
+
+    :returns: domain object with the updated information
+
+    :raises ValueError: raised when the domain is already assigned to
+    the organization
+    """
+
```

Comment:
In GrimoireLab when there is a multi-line docstring we don't leave an empty line after it, but when the docstring is a single line we add it.
```suggestion

```